### DISCUSSION TITLE
Applying text center style in the component.css file does not work for a fluent data grid PropertyColumn

### DISCUSSION THREAD

**Discussion by jobou363** [microsoft/fluentui-blazor#1739] [General]
I'm struggling to understand how I just want to center text inside my grid column by using the component css file.

The text is not centered in this picture : 
![image](https://github.com/microsoft/fluentui-blazor/assets/240580/fda9f620-6e7f-46ec-afda-aa13950bfafa)

I want it like this : 
![image](https://github.com/microsoft/fluentui-blazor/assets/240580/5feb119b-8467-4e41-8d14-4ab614091379)

I found 2 way of doing this that work but i try to understand why it's not working with the css in the component.

1) it works using <PropertyColumn **Style="align-self: center"** 
2) it works also doing a class in the global app.css and using it PropertyColumn **Class="align-self-center"**
`  .align-self-center {
      align-self: center;
  }`
3) but if i try to use the same css class PropertyColumn **Class="align-self-center2"** but by defining it in the Component.razor.css instead the style is not applied to the PropertyColumn 

`  .align-self-center2 {
      align-self: center;
  }`

In chrome i found the element to have an id "[b-w6qdxfylwy]" so is there is something i don't do correctly in the css?
`
fluent-data-grid-cell[b-w6qdxfylwy] {
    text-overflow: ellipsis;
}
`

The code of the complete fluent data grid.

` <FluentDataGrid GridTemplateColumns="0.5fr 0.5fr 1fr" Class="mt-3" Style="width: 480px" Context="GridContext" ItemsProvider="@GridItemsProvider" TGridItem="PlanDto" GenerateHeader="GenerateHeaderOption.Sticky">

     <TemplateColumn Align="@Align.Center" >
         <FluentButton IconStart="@(new Icons.Regular.Size20.Link())" OnClick="@(() => NavigateToWithParameter(TaskAction, context.PlanEntityId))">
             @TaskAction?.ActionProperties.GetDescription()
         </FluentButton>
     </TemplateColumn>

     <PropertyColumn Class="align-self-center" Title="@Localization.Abreviation"  Property="@(p => p.Abbreviation)" Align="Align.Start" Sortable="true"/>
     <PropertyColumn Class="align-self-center2" Title="@Localization.Description" Property="@(p => p.Description)" Align="Align.Start" Sortable="true"/>

 </FluentDataGrid>`

The component css file
.align-self-center2 {
    align-self: center !important;
}

**Comment by vnbaaij** [comment]
It doesn't work because of the scoping that happens when you use isolated CSS. Nor really a Datagrid or Fluent issue. It's a general Blazor consequence. Learn more about that at https://learn.microsoft.com/en-us/aspnet/core/blazor/components/css-isolation?view=aspnetcore-8.0
